twarep/electrifly
# ML Prescription Models

## Checklist
**Note:** Put an x into the box like this [x] to check these off

## What I Did
What files git you edit and why/for what purpose?
- flight_querying.py has updated queries used in ML models
- created 5 models (linear regression, ridge regression, neural network, random forest, XGBoost)

## How I Did it?
What packages did I install to accomplish this goal? What libraries did I need that the team might not know?
Write anything in this section that is technically related that everyone on the team might not know or is not
trivial.
- sklearn, torch, etc. 

## How to Test?
Manual testing? Automated testing? E2E testing? Smoke tests?
- run requirements.txt (make sure you have torch installed) 
- run all the cells in these jupyter notebook files: 
![Screenshot 2024-02-05 at 8 18 49 PM](https://github.com/twarep/electrifly/assets/71115362/3e5b9f39-4a05-4d10-8aad-3d2821ea64fb)

### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
